lib/vanagon/platform/dsl.rb

```diff
@@ -198,6 +198,20 @@ def build_hosts(*args)
       # Because single vs plural is annoying to remember
       alias_method :build_host, :build_hosts
 
+      # Set a flag whether this platform is cross-compiled
+      #
+      # @param type [Boolean] true if this platform is cross-compiled
+      def cross_compiled(flag)
```

Comment:
In theory, project files should be treated like a DSL and not like raw Ruby. In practice, they're straight up raw Ruby. I'm not opposed to making this a simple thing that gets called in a project like this:

```ruby
project "puppet-agent" do |proj|
  cross_compiled
  platform = proj.get_platform
```